tests/test_playbooks/vars/subnet.yml

```diff
@@ -4,7 +4,6 @@ subnet_orgs:
 - Test Org2
 subnet_locs:
 - Foo
-- Foo/Baz
```

Comment:
Why?

Reply:
I guess, we look up the locations in the wrong way. I guess, i should fix that.

Reply:
If we want to search locations by title, i urge you to ack/merge #332 .